packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -755,20 +755,20 @@ class PhysicalModelLayer extends ContainerLayer {
   /// Creates a composited layer that uses a physical model to producing
   /// lighting effects.
   ///
-  /// The [clipRRect], [elevation], and [color] arguments must not be null.
+  /// The [clipPath], [elevation], and [color] arguments must not be null.
   PhysicalModelLayer({
```

Comment:
rename this one to match the name you end up with on the engine side

Reply:
done